### ISSUE TITLE
Filter button: Announce the number of filters

### ISSUE THREAD

**Issue by marvieqa** [CodeForPhilly/clean-and-green-philly#654]
## Description

When filters are applied, we put the number of filters in the filter button. This number should be announced by screen readers to denote that there are filters currently applied. 

## Recommendation

- announce the number of filters applied . Ex. "Filter button, x filters applied"

**Related ticket(s)**
https://github.com/CodeForPhilly/clean-and-green-philly/issues/625

**Comment by jgrimes86** [comment]
@marvieqa or @Moylena could I work on this issue and issue #627 ?

**Comment by bacitracin** [comment]
@jgrimes86 assigned!
